scripts/scr_enemy_ai_e/scr_enemy_ai_e.gml

```diff
@@ -367,8 +367,12 @@ function scr_enemy_ai_e() {
 	run=0;force=1;beetle=0;chaos_meeting=0;
 
 	repeat(4){run+=1;force=1;
+		if (p_player[run]>0){
+	   		var forces_list=scr_count_forces(name, run, true,true)
+	   		var force_count = forces_list[0]+forces_list[1];
+		}
 
-	    if (p_player[run]>0){
+	    if (p_player[run]>0 && force_count>0){
```

Comment:
`force_count>0` will throw an error, because you declared `force_count` in `if(player[run]>0{...}`, which will get disposed before this if is reached